app/views/nominations/review.html.erb

```diff
@@ -0,0 +1,55 @@
+<% content_for :title, "Resend nomination: confirm school" %>
+<% content_for :before_content, govuk_back_link(
+  text: "Back",
+  href: choose_location_nominations_path)
+%>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Confirm your details</h1>
+    <p class="govuk-body govuk-!-margin-bottom-7">
+      An email will be sent to your school with a link to nominate an induction tutor.
+    </p>
+
+    <div class="govuk-!-margin-bottom-9">
+      <h2 class="govuk-heading-m govuk-!-margin-bottom-1">School details</h2>
+      <dl class="govuk-summary-list">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            Name
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= @nomination_form.school.name %>
+          </dd>
+        </div>
+
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            URN
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= @nomination_form.school.urn %>
+          </dd>
+        </div>
+
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            Address
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= simple_format @nomination_form.school.full_address, class: "govuk-body" %>
+          </dd>
+        </div>
+      </dl>
+
+      <%= govuk_link_to "Change school", choose_location_nominations_path(continue: true), class: "govuk-link--no-visited-state" %>
+    </div>
+
+    <%= form_for :confirmation, url: review_nominations_path, method: :post do |f| %>
+      <%= f.hidden_field :answers_checked, value: true %>
+      <%= f.govuk_submit "Confirm" %>
+    <% end %>
+  </div>
+</div>
+
+
```

Comment:
Michal - empty space? This shall not pass as Gandalf the Gray had once said.

Reply:
Ahh, he could spot it in others' code, but not his own 😂